app/lib/actions/pulp3/repository/save_version.rb

```diff
@@ -3,35 +3,39 @@ module Pulp3
     module Repository
       class SaveVersion < Pulp3::Abstract
         def plan(repository, options)
-          plan_self(:repository_id => repository.id, :tasks => options[:tasks], :repository_details => options[:repository_details])
+          fail "Cannot accept tasks and repository_details into Save Version." if options[:tasks].present? && options[:repository_details].present?
+          plan_self(:repository_id => repository.id, :tasks => options[:tasks], :repository_details => options[:repository_details], :force_fetch_version => options.fetch(:force_fetch_version, false))
         end
```

Comment:
How are we using this? Is there a way for users to set force_fetch_version to true?

Reply:
updated to pass this in through the pulp3 repo sync orchestration action

also added a test that tests this functionality.  You can test manually too:

```
> Katello::Repository.first.version_href
=> "/pulp/api/v3/repositories/rpm/rpm/0d068318-16a4-4f1f-bc82-6a4fc296ad5e/versions/3/"
```
Find a repo and reset its version_href to zero (note you may want to use .find(id) here if you have more than 1 repo.
```
> Katello::Repository.first.update_attributes!(version_href:  "/pulp/api/v3/repositories/rpm/rpm/0d068318-16a4-4f1f-bc82-6a4fc296ad5e/versions/0/")
```

Re-index the repo at version zero:

```
> Katello::Repository.first.index_content
> Katello::Repository.first.rpms
0
```

Now do an advanced sync, skipping the metadata check.  You should see the version href go back to the newest version, and the proper content get indexed from that latest version. 

This workflow simulates what happens if a repo gets updated outside of katello (or katello fails to index it properly), and the user performs this action to correct it